.github/workflows/open-datadog-agent-pr.yml

```diff
@@ -5,10 +5,17 @@ on:
     types:
       - closed
 
+permissions: {}
+
 jobs:
   open_bump_pr:
     if: github.event.pull_request.merged == true && !contains(github.event.pull_request.labels.*.name, 'no-auto-bump') && github.ref == 'refs/heads/master'
     runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
I'm not sure this is needed, since we create a token for a github app which is the one actually submitting the pull request. The workflow's GITHUB_TOKEN doesn't seem to be involved.

Reply:
That is true, thanks for spotting it

Reply:
It might be worth doing a test run just in case, to make sure the workflow does indeed do what's expected with those permissions.

Reply:
Done by bypassing the rules on this branch on [3856459](https://github.com/DataDog/omnibus-software/pull/539/commits/385645965e5bff636bde4be83ed8c8a35f5e3091)

The job https://github.com/DataDog/omnibus-software/actions/runs/10352697350/job/28653880233?pr=539
Created the PR
https://github.com/DataDog/datadog-agent/pull/28394